.golangci.yml

```diff
@@ -112,6 +112,11 @@ linters:
     - godox
     - gocognit
     - gomnd
+    - nolintlint # TODO: fix & enable me
+    - testpackage # TODO: fix & enable me
+    - goerr113
+    - nestif
```

Comment:
Are these useful?

Reply:
doesn't looks like that, or they're far away from our coding style.

`goerr113` prohibits `fmt.Errrof`

`nestif` seems to complain a lot around our code being too complicated